doc/source/methods/ALE.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<p>I would abide by the rules of academic objectivity in the 2nd paragraph and simply say.</p><p> </p><p>_ALE addresses some key shortcomings of Partial Dependence Plots, a popular method of estimating first order feature effects. We discuss these limitiations and motivate ALE after presenting the method usage._</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>